catalog/catalog-app/src/main/resources/features.xml

```diff
@@ -443,15 +443,29 @@
         </bundle>
     </feature>
 
+    <feature name="catalog-metacardbackup-storage-api" install="manual" version="${project.version"
```

Comment:
nit : the other feature has a hyphen between metacard and backup; looks odd when presented in the features list of the Admin UI